src/sky.cpp

```diff
@@ -202,15 +202,14 @@ void Sky::render()
 
 		driver->setMaterial(m_materials[1]);
 
-		video::SColor cloudyfogcolor = m_bgcolor;
```

Comment:
This is best kept as it makes the code more readable, 'background color' is confusing in this file as it is unintuitive to think of that as haze colour. but 'cloudyfogcolor' is clear in the usage below.